internal/cli/root.go

```diff
@@ -10,7 +10,10 @@ import (
 
 const AppName = "stego"
 
-var Version = "0.0.0-dev"
+var (
+	Version = "0.0.0-dev"
+	Verbose bool
```

Comment:
I would keep it unexported, but it looks good anyway.

Reply:
should I make it unexported then? Sorry, it's not clear

Reply:
Yes please! :+1:

Reply:
done

Reply:
Thank you!